scripts/in_container/_in_container_utils.sh

```diff
@@ -263,14 +263,16 @@ function install_airflow_from_wheel() {
     echo
     echo "Install airflow wheel package from dist"
     echo
-    pip install /dist/apache_airflow-*.whl >"${OUT_FILE_PRINTED_ON_ERROR}" 2>&1
+    pip install /dist/apache_airflow-*.whl --no-deps >"${OUT_FILE_PRINTED_ON_ERROR}" 2>&1
```

Comment:
Can you explain why you are adding the `--no-deps` flag here?

Reply:
Yep. I can add a comment, but this is easy to explain. By that time all the dependencies are already installed (we remove only the packages (airflow + airflow-providers) that were installed from sources, but their dependencies are already installed as needed as we installed 'devel_ci' extra. That's why we just need to install the packages, build and nothing else.

Reply:
Also we do not want to set constraints here as we know exactly what we are installing - those that are in the .whl files.